src/client/dfuse/pil4dfs/int_dfs.c

```diff
@@ -876,6 +893,26 @@ is_path_start_with_daos(const char *path, char *pool, char *cont, char **rel_pat
 	return true;
 }
 
+static void
+child_hdlr(void)
+{
+	int rc;
+
+	/* daos is not initialized yet */
+	if (!daos_inited)
+		return;
+
+	daos_eq_lib_reset_after_fork();
```

Comment:
[minor] daos_eq_lib_reset_after_fork() might fail and return failure is ignored.

Reply:
Don't we get coverity warnings for these? Meaning, wouldn't this eventually get flagged?

Reply:
coverity scan is done for master, 2.2, and 2.4 branch. It does not scan PRs. It will generate a warning after PR is landed as you said. I am not sure whether coverity will generate a warning about this one since this is a new function. Normally coverity reports an issue like "The return code of fun_foo() is not checked. It is checked in other cases (e.g., 20 out of 26)."